sdks/python/pom.xml

```diff
@@ -56,6 +56,41 @@
     </pluginManagement>
     <plugins>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-antrun-plugin</artifactId>
+          <executions>
+            <execution>
+              <id>copy-project-files</id>
+              <phase>prepare-package</phase>
+              <goals>
+                <goal>run</goal>
+              </goals>
+              <configuration>
+                <tasks>
+                  <copy file="../../LICENSE" tofile="${basedir}/LICENSE"/>
+                  <copy file="../../NOTICE" tofile="${basedir}/NOTICE"/>
+                </tasks>
+              </configuration>
+            </execution>
+          </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-antrun-plugin</artifactId>
+          <executions>
+            <execution>
+              <id>copy-project-files-clean</id>
+              <phase>clean</phase>
```

Comment:
This is much better served by configuring the maven-clean-plugin directly, as opposed to creating a new execution to delete a file.